### DISCUSSION TITLE
Having trouble applying a transition to the `width` property on an element using `vw`?

### DISCUSSION THREAD

**Discussion by anmpog** [system-ui/theme-ui#2354] [Q&A] [answered]
Deleted as I figured this out and it was entirely error on my end.

**Comment by lachlanjc** [comment, marked as answer]
Glad you got it sorted out! Animating the `width` property generally isn't a good idea in CSS, as it'll cause layout recalculations. Here's one article I like about this topic: https://pqina.nl/blog/animating-width-and-height-without-the-squish-effect/
